.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish artifacts
```

Comment:
These are unrelated jobs with different triggers, aren't they? Can we separate them into two different workflows?

Reply:
we can but as this repo only has charts so thought of combining them and using if condition as running validate charts on release is fine anyways.

Reply:
Should I separate them?

Reply:
I think so - there's very little duplication - just the triggers themselves. We also want to make sure we run validate on push (which should make it unnecessary on release). It also saves us from having the publish job show up as skipped on every PR and allows us to keep this naming more consistent: `Publish artifacts / validate-helm-charts ` -> `Pull Request/ validate-helm-charts `

Reply:
cool

Reply:
addressed